src/components/pages/Triggers/utils.ts

```diff
@@ -1,24 +1,36 @@
-import {AddTriggerOption} from '@models/triggers';
+import {Option} from '@models/form';
+import {TestTriggerSelector} from '@models/triggers';
 
-export const addTriggerOptions: AddTriggerOption[] = [
-  {
-    key: 'label-label',
-    label: 'Labels to Labels',
-    description: 'Identify your cluster and testkube resources by label',
-  },
-  {
-    key: 'name-label',
-    label: 'Name to Labels',
-    description: 'Identify your cluster resource by name, your testkube resources by label',
-  },
-  {
-    key: 'name-name',
-    label: 'Name to Name',
-    description: 'Identify your cluster and testkube resource by name',
-  },
-  {
-    key: 'label-name',
-    label: 'Labels to Name',
-    description: 'Identify your cluster resources by labels, your testkube resource by name',
-  },
-];
+import {decomposeLabels} from '@molecules/LabelsSelect/utils';
+
+export const getResourceIdentifierSelector = (
+  formValue: string | readonly Option[],
+  appNamespace: string
+): TestTriggerSelector => {
+  if (typeof formValue === 'string') {
+    if (formValue?.includes('/')) {
```

Comment:
There is already known that the `formValue` is `string`:
```suggestion
    if (formValue.includes('/')) {
```